### ISSUE TITLE
The Language bar show invalid output

### ISSUE THREAD

**Issue by ahmednss** [agalwood/Motrix#1334]
**Describe the bug**
If you attempt to change language in `preference > basic` without first clicking the save and apply button, then switch to another menu, such as `preferences > advanced`, and then back. The old language will be displayed in the Language bar with the selected language already applied.

**To Reproduce**
1. Go to `preference > basic`
2. Click on Language bar.
3. select arabica Language.
4. Go to `preference > advanced`.
5. Go back to `preference > basic`.
6. Scroll down to Language bar.
7. See error

**Expected behavior**
Only the right language will be displayed in the language bar; the language won't change until I press the save and apply button.

**Screenshots**
![Screenshot 2022-11-18 043329](https://user-images.githubusercontent.com/62723180/202603833-6662e9d3-d222-42c0-bc6c-6ae0d07cb023.png)

**Environment (please complete the following information):**
 - OS & Version: Windows
 - Version: Windows 11
 - Motrix Version: v1.6.11

**Comment by CDmking** [comment]
Yes, it can only do that. After making changes, you can only press _Save & Apply_ to make the settings take effect.

**Comment by agalwood** [comment]
Closing as legacy-v1.

Motrix v2 has replaced the legacy 1.x codebase, so this issue is no longer tracked. If it still reproduces with the latest Motrix v2 beta, please open a new issue with the current Motrix version, OS, reproduction steps, and logs.

Motrix v2 已替换 1.x 旧架构，因此不再跟踪此 issue。若问题仍能在最新 Motrix v2 beta 复现，请重新提交 issue，并附上当前版本、系统、复现步骤和日志。
